src/main/kotlin/io/snyk/plugin/cli/Vulnerability.kt

```diff
@@ -33,4 +33,12 @@ class Vulnerability {
             ""
         }
     }
+
+    fun getSeverityIndex(): Int =
+        when (severity) {
+            "high" -> 3
+            "medium" -> 2
+            "low" -> 1
+            else -> 0
```

Comment:
I see we use it for sorting later. How about using comparator for tree nodes? Issue I see atm: this code cannot handle other issues e.g. `critical` if this will come later.

Reply:
Don't we need to handle the case with `critical` severity anyway inside those comparator? 
Then why to add one more level of abstraction here?

Reply:
left //todo there for the future